docs/hub/datasets-image.md

```diff
@@ -213,4 +218,4 @@ dataset_info:
     dtype: string
 ```
 
-Alternatively, Parquet files with Image data can be created using the `datasets` library by setting the column type to `Image()` and using the `.to_parquet(...)` method or `.push_to_hub(...)`. You can find a guide on loading image datasets in `datasets` [here](/docs/datasets/image_load).
+Note that Parquet is recommended for small images (<1MB per image) and small row groups (100 rows per row group, which is what `datasets` uses for images). For larger images it is recommended to use the WebDataset format, or to share the original image files (optionally with metadata files).
```

Comment:
Just to clarify, does the guidance here say that parquet makes sense only if you have small images AND small row groups? I suspect more people will default to original image files so maybe we also need to point to the repository limits page too in that case?

Reply:
yup correct, I'll add a link to the repo limits